yt_dlp/extractor/iwara.py

```diff
@@ -73,8 +78,17 @@ def _real_extract(self, url):
             r'<title>([^<]+)</title>', webpage, 'title'), ' | Iwara')
 
         thumbnail = self._html_search_regex(
-            r'<video[^>]+id=[\'"]video-player[\'"][^>]+poster=[\'"]([^\'"]+)',
-            webpage, 'thumbnail', default=None)
+            r'poster=[\'"]([^\'"]+)', webpage, 'thumbnail', default=None)
+
+        uploader = self._html_search_regex(
+            r'class="username">([^<]+)', webpage, 'uploader', fatal=False)
+
+        upload_date = self._html_search_regex(
+            r'作成日:([^\s]+)', webpage, 'upload_date', fatal=False).replace("-", "")
+
+        description = self._search_regex(
+            r'<p>(.+?(?=</div))', webpage, 'description', fatal=False,
+            flags=re.DOTALL).strip()
```

Comment:
```suggestion
        description = strip_or_none(self._search_regex(
            r'<p>(.+?(?=</div))', webpage, 'description', fatal=False,
            flags=re.DOTALL))
```
(and add import from utils)
So it doesn't raise an error if no description is extracted.